pkgpanda/http/config.py

```diff
@@ -0,0 +1,12 @@
+DCOS_ROOT = '/opt/mesosphere/'
```

Comment:
This makes the `pkgpanda.cli` docstring less useful, but it's the best I can come up with: https://github.com/dcos/dcos/pull/91/commits/f7d7a475604178448096812acd5be85d1c23467a